WORKSPACE

```diff
@@ -18,10 +13,34 @@ load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_
 rules_proto_dependencies()
 rules_proto_toolchains()
 
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "yarn_install")
+load("@aspect_rules_js//js:repositories.bzl", "rules_js_dependencies")
+rules_js_dependencies()
+
+load("@aspect_rules_ts//ts:repositories.bzl", "rules_ts_dependencies")
+rules_ts_dependencies(
+    ts_version_from = "//:package.json",
+)
 
-yarn_install(
+# Fetch and register node, if you haven't already
+load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
+nodejs_register_toolchains(
+    name = "nodejs",
+    # This is the version of Node.js that would run the generator, it's unrelated to the versions supported by the generated libraries
+    node_version = "18.12.1", # https://github.com/bazelbuild/rules_nodejs/blob/stable/nodejs/private/node_versions.bzl 
```

Comment:
Nit: can/should this be referenced from a single spot such that when you want to update this value, the other spots (e.g. CI, package.json or whatever) are also updated?

Reply:
Added a constant in `repositories.bzl` and imporing it from the workspace file. The same import will be added in the googleapis `WORKSPACE`.